@here/harp-datasource-protocol/test/StyleSetEvaluatorTest.ts

```diff
@@ -288,6 +288,25 @@ describe("StyleSetEvaluator", function () {
             })();
 
             assert.equal(techniquesTileA.length, 2);
+
+            const techniquesTileB = (() => {
+                const ev = new StyleSetEvaluator({ styleSet: texturedStyle });
+                ev.getMatchingTechniques(new MapEnv({ kind: "park", map: { buffer: buffer2 } }));
```

Comment:
Please also add a test where the buffer: both point to `buffer1` and the length is 1

Reply:
done